src/perfrecord.cpp

```diff
@@ -172,6 +172,13 @@ void PerfRecord::startRecording(const QStringList& perfOptions, const QString& o
     }
     m_perfRecordProcess = new QProcess(this);
     m_perfRecordProcess->setProcessChannelMode(QProcess::MergedChannels);
+    auto e = QProcessEnvironment::systemEnvironment();
+    foreach(const QString &str, env){
+        QStringList var = str.split(QChar::fromLatin1('='));
```

Comment:
const auto var = str.split(QLatin1Char('='));

Reply:
hm no, that's actually not enough, when the (string) value contains another = then we don't want to split again, so this should find the first `=` via indexOf and then pass the first and last part to `e.insert`